### DISCUSSION TITLE
[Data Importer] How to use with Bunq?

### DISCUSSION THREAD

**Discussion by BobW5** [firefly-iii/firefly-iii#6001] [Firefly III]
With the Bunq Importer discontinued, can anyone explain me how to use the new Data Importer together with Bunq?

**Comment by JC5** [comment]
Salt Edge supports bunq: https://docs.firefly-iii.org/data-importer/install/nordigen-spectre/

**Reply by BobW5** [reply]
Hi James,

I tried following those instructions and got below reply:

Hello Bob,



Thank you for submitting your request for "Test" status access. 



Before proceeding, I would like to let you know that Salt Edge does not offer any personal use payment plans, Salt Edge is a B2B oriented company, thus kindly let me know whether you are planning to use the developed application and the Spectre API functionality in a commercial way.



In case the project is solely for personal use, I would like to suggest you to use our partners platform: FireFly III



https://www.firefly-iii.org/



On their website you will find a detailed guide on how to get unlimited free access for personal use with Salt Edge API.



Looking forward to your feedback.



Kind regards,

**Reply by BobW5** [reply]
I think somehow I managed to mess up..

**Reply by JC5** [reply]
E-mail them back and say "I use Firefly III". You can point to this thread.

**Reply by BobW5** [reply]
Thank you will try this 👍
